rpc/tests/utils/client.rs

```diff
@@ -54,12 +53,13 @@ pub async fn new_test_client(auth_level: AuthLevel) -> Result<WsClient> {
 }
 
 // This can be used if you want to inspect the requests from `mitmproxy`.
-pub async fn new_test_client_http(auth_level: AuthLevel) -> Result<HttpClient> {
+pub async fn new_test_client_http(auth_level: AuthLevel) -> Result<Client> {
```

Comment:
Actually we can remove this now. `Client` will take the decision based on `CELESTIA_RPC_URL` env var.